microservices/jwt-audit-analytics-writer/dev/values.yaml

```
@@ -0,0 +1,19 @@
+name: "jwt-audit-analytics-writer"
```

```suggestion
name: "interop-be-audit-analytics-writer"
```
Not 100% sure we want to keep this prefix in the future, but for now let's use it for consistency

---

My bad, I removed that part by accident